### ISSUE TITLE
Would be nice if there was a filter for low precision

### ISSUE THREAD

**Issue by rvosa** [ropensci/CoordinateCleaner#33]
I think this is a great package, and what I want is easy to implement in some other way so I'm just being lazy - but it would be nice if there was a `cc_precision` that filters out coordinates with low precision (e.g. less than 2 decimal places or whatever).

Cheers,

Rutger

**Comment by jocelynpender** [comment]
I agree! This would be great if it were to be included in the package. It's easy to do.. but it is also a natural fit for the package (e.g., it could be included in `clean_coordinates()` by default)
